test/unit/cross_lane_ops_test/detail/cross_lane_ops.hpp

```diff
@@ -56,11 +57,10 @@ namespace rocwmma
         dim3 gridDim() const final
         {
             // Need to address the input array as 32b elements per thread.
-            auto x
-                = dim3(static_cast<uint32_t>(roundf(static_cast<float32_t>(sizeof(DataT))
-                                                    / static_cast<float32_t>(sizeof(uint32_t))
-                                                    * static_cast<float32_t>(Base::mM * Base::mN)))
-                       / Base::mTBlockX);
+            auto x = dim3(static_cast<uint32_t>(
+                              roundf(static_cast<float32_t>(Base::mM * Base::mN)
+                                     / static_cast<float32_t>(PackTraits<DataT>::PackRatio)))
+                          / Base::mTBlockX);
```

Comment:
The change from sizeof(DataT) / sizeof(uint32_t) -> PackRatio leads to a different number of elements for f64 as the PackRatio for f64 is 1.

Reply:
yeah, that's correct. And `sizeof(double) / sizeof(uint32_t) == 2` was a bug. 

In rocWMMA, multiple data elements can be processed by one thread if they can be packed into one uint32_t. And a double element is processed by one thread in a static loop as well.

So `sizeof(double) / sizeof(uint32_t) == 2` resulted in twice as many threads as expected and led to a crash issue.